docs/host-page-integration.md

```diff
@@ -0,0 +1,29 @@
+# Interacting with the sidebar
+
+## Show the sidebar:
+
+It is possible to show the Hypothesis sidebar from a third party page
```

Comment:
"It is possible to..." is probably not the best way to start documentation for the section that is titled with the action they are wanting to do. Perhaps formatting it like this:

```
## Show the sidebar:

If you need to have a custom trigger on your third party page to bring up the sidebar, add `data-hypothesis-trigger` to any element that you want to enable. Clicking that element will cause the sidebar to open. Note, however, subsequent clicks do not hide the sidebar.

For example:

.... your example.
```

Where you are explaining the api with the question itself and any additional information can be added for caveats and concerns.